PyTorch_dcgan_lsun/README.md

```diff
@@ -0,0 +1,64 @@
+# PyTorch - DCGAN
+
```

Comment:
Missing requirements, requirements.txt installation command?

Reply:
the requirements are in the original repo, and since it needs to be cloned either way I don't see a point in adding them here too

Reply:
Right, my bad.
Maybe we should mention the requirements installation in README though.